Sources/String + CleanedString.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2015 Stephen Radford. All rights reserved.
 //
 
+import Foundation
```

Comment:
Since this file uses `NSCharacterSet` a foundation type, `Foundation` should be imported.

This may not be necessary while building the source in the context of an iOS application because `Foundation` is either implicitly imported or available via a pch/bridge header.

But, without this change, Downpour will not build from Swift Package Manager.